### ISSUE TITLE
Looking for more awesome people to contribute

### ISSUE THREAD

**Issue by sichvoge** [raml2html/raml2obj#24]
RAML2OBJ powers one of the most successful RAML projects with over 24k downloads in the last month, [RAML2HTML](https://github.com/raml2html/raml2html). With the latest changes, supporting RAML 1.0, we are currently looking for awesome people that help us to maintain and keep it as successful as it is.

Kevin, the brain behind that project, will move on to tackle other projects. Hence, it’s your time to jump in and lift his baby to the next stage.

Please let us know if you’d like to help and support by commenting down on this issue. If not, but maybe you know someone, tweet - get out the word that we are looking for contributors.

Contributors should do either one or all (is up to you and your time):

Review and answer issues
Review open PRs and merge
Code Code Code
Help us to keep this amazing project alive and well.

What about you @MichaelIT or @naterkane ;)

**Comment by MichaelIT** [comment]
yes, I like to help this project, but I am lack of experience for contribution on github.I will try my best to do it if u give me a try.

**Comment by kevinrenskers** [comment]
Awesome! The best way to get started is simply by picking up open issues in raml2obj and raml2html. Fix some bugs, add missing features, send pull requests, help to answer questions. After that we can give you push access to the repos.

**Comment by MichaelIT** [comment]
ok, I will be back after Chinese New Year holiday.

**Comment by sichvoge** [comment]
Super awesome! Thanks @MichaelIT, looking forward for your contribution.

**Comment by Raederle** [comment]
I'm definitely interested in contributing.  I added a minor fix to the raml2html default-template and will look through the issues list to see if there are other places I can contribute.

**Comment by kevinrenskers** [comment]
Awesome! We just had 2 issues reported related to the new 7.0.0 release, maybe you could look into those?
